mx-tester.yml

```diff
@@ -6,7 +6,7 @@ up:
     # Wait until postgresql is ready
     - until psql postgres://mjolnir-tester:mjolnir-test@127.0.0.1:8083/mjolnir-test-db -c ""; do echo "Waiting for psql..."; sleep 1s; done
     # Launch the reverse proxy, listening for connections *only* on the local host.
-    - docker run --rm --network host --name mjolnir-test-reverse-proxy -p 127.0.0.1:8081:80 -v $MX_TEST_CWD/test/nginx.conf:/etc/nginx/nginx.conf:ro -d nginx
```

Comment:
what is the story behind this?

Reply:
Its fully ignored as a parameter. This just happened to be cleaned up along figuring this out. `--network host` causes `-p` to be fully ignored :) It even prints a warning when it is present.